src/library/Uno.Material/Controls/Card.cs

```diff
@@ -13,24 +14,44 @@ public partial class Card : Control
 		private const string __openedState = "Opened";
```

Comment:
I know you didn't touch it, but can you change these two to $`{State}Name`?

Reply:
I will make the change, thanks !